### PR TITLE
Update outdated content in README.md

### PR DESCRIPTION
**Pull request by chandankumar4**
Fixes: #21

### CODE REVIEW

**Comment by chandankumar4** [comment]
Lint check is expected to fail as there is no Chart version upgrade due to only change in README.md file. I have added it as ignore path which should works with next PRs

File: charts/numaflow/README.md [hunk 1]
```diff
@@ -47,14 +47,14 @@ Output:
...
 STATUS: deployed
 REVISION: 1
 TEST SUITE: None
 NOTES:
 1. Get the application URL by running these commands:
-  export POD_NAME=$(kubectl get pods --namespace numaflow-system -l "app.kubernetes.io/name=numaflow-server" -o jsonpath="{.items[0].metadata.name}")
+  export POD_NAME=$(kubectl get pods --namespace numaflow-system -l "app.kubernetes.io/name=numaflow-ux" -o jsonpath="{.items[0].metadata.name}")
   export CONTAINER_PORT=$(kubectl get pod --namespace numaflow-system $POD_NAME -o jsonpath="{.spec.containers[0].ports[0].containerPort}")
   kubectl --namespace numaflow-system port-forward $POD_NAME $CONTAINER_PORT
   echo "Visit http://127.0.0.1:$CONTAINER_PORT to use your application"
```
Comment (vigith) [inline charts/numaflow/README.md:60]: is this a better way? 

```
k port-forward -n numaflow-system svc/numaflow-server 8443 
```

Comment (chandankumar4) [reply charts/numaflow/README.md:60]: yaa, It could be, if we wanted to hardcode the name and port

Comment (chandankumar4) [reply charts/numaflow/README.md]: Updated

**Review by vigith — APPROVED** [review]

**Pull request merged**